hack/add/analysis.sh

```
@@ -5,14 +5,14 @@ set -e
 host="${HOST:-localhost:8080}"
 appId="${1:-0}"
 nRuleSet="${2:-10}"
-nIssue="${3:-10}"
+nInsight="${3:-10}"
 nIncident="${4:-25}"
 tmp=/tmp/${self}-${pid}
```

Shellcheck (0.10.0)

[warning] self is referenced but not assigned.

(SC2154)